civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -6394,3 +6401,107 @@ function civihr_employee_portal_node_export_import_alter(&$nodes, $format, $save
     WebformTransferService::preImport($node);
   }
 }
+
+/**
+ * Creates the default reports configuratio in the database
+ */
+function _civihr_employee_portal_create_default_reports_configuration() {
+  $configurations = [
+    [
+      'report_name' => 'undefined',
```

Comment:
Shouldn't this belong to the people report?

Reply:
This shouldn't have been included